tests/test_cli_data.py

```diff
@@ -67,78 +80,65 @@ def test_data(config):
     assert "List data in managed data on Deep Origin" in stdout, "Unexpected output"
 
 
-def test_describe_file(config):
+@pytest.mark.parametrize("json_option", JSON_OPTIONS)
+def test_describe_file(config, json_option):
     file_id = config["file"]["id"]
 
     stdout = _run_cli_command(
-        ["data", "describe-file", file_id],
+        ["data", "describe", file_id] + json_option,
         config["client"],
     )
-    assert file_id in stdout, "Expected to see file_id in output"
+
+    if json_option == ["--json"]:
+        data = _check_json(stdout)
+        assert data["id"] == file_id, "Expected file ID to match"
 
 
-def test_describe_row(config):
+@pytest.mark.parametrize("option", JSON_OPTIONS)
+def test_describe_row(config, option):
     row_id = config["rows"][0]
 
     stdout = _run_cli_command(
-        ["data", "describe-row", row_id],
+        ["data", "describe", row_id] + option,
         config["client"],
     )
     assert row_id in stdout, "Expected to see row_id in output"
 
-    # try JSON output
-    stdout = _run_cli_command(
-        ["data", "describe-row", row_id, "--json"],
-        config["client"],
-    )
-
     # check that we can parse into JSON
-    data = json.loads(stdout)
-
-    assert isinstance(data, dict), "Expected data to be a dictionary"
-
-    assert data["id"] == row_id, "Expected row ID to match"
+    if option == ["--json"]:
+        data = _check_json(stdout)
 
+        assert data["id"] == row_id, "Expected row ID to match"
 
-def test_list_rows(config):
-    db_id = config["databases"][0]
 
+@pytest.mark.parametrize("json_option", JSON_OPTIONS)
+@pytest.mark.parametrize("list_option", LIST_OPTIONS)
```

Comment:
the list command has a bunch of options, and we use a single function to test all of them